src/FrontendAccountCreation.Core.UnitTests/AccountMapperTests.cs

```diff
@@ -264,7 +333,7 @@ public void testCreateAccountModel_DeclarationProperties_ShouldReturnSetValues()
                 Company = new Company
                 {
                     AccountCreatedOn = DateTime.Now,
-                    BusinessAddress = new Addresses.Address { BuildingName = "building name" },
+                    BusinessAddress = new Address { BuildingName = "building name" },
```

Comment:
This test flakiness is why I always use AutoFixture. 
You may need additional asserts, but dont need to create models like this :)